packaging/amazon-linux-ami/ecs-init.spec

```diff
@@ -255,6 +255,10 @@ if [ -e %{running_semaphore} ]; then
 fi
 %endif
 
+%changelog
+* Fri Jan 4 2019 Eric Sun <yuzhusun@amazon.com> - 1.24.0-1
+- Cache Agent version 1.24.0
+
 %changelog
```

Comment:
This `%changelog` line needs to be removed.

Reply:
Removed